aten/src/ATen/function_wrapper.py

```diff
@@ -233,15 +233,23 @@ def __init__(self, reason):
     'THGenerator*':
         CodeTemplate(
             'check_generator<${Backend}Generator>(${arg_name}, &context->defaultGenerator(backend()))'),
-    'THSize*': CodeTemplate('THLongStorageView::makeFromSize(${arg_name})'),
-    'THStride*': CodeTemplate('THLongStorageView::makeFromStride(${arg_name}, ${noelem_to_empty})'),
+    # This is a cast done via direct-construction
+    'THSize*': CodeTemplate('THLongStorageView ${arg_name}_(${arg_name}, THLongStorageViewKind::SIZE);'),
+    # This is a cast done via direct-construction
+    'THStride*':
+        CodeTemplate(
+            'THLongStorageView ${arg_name}_(${arg_name}, '
```

Comment:
nit: we use this naming pattern already sometimes, should we use something else to guarantee it won't conflict?